core/src/components/text-area/text-area.stories.tsx

```diff
@@ -1,15 +1,22 @@
-import { storiesOf } from "@storybook/react";
+import { _Story } from "../../_story";
 import { DivPx } from "../div/div";
 import { TextArea } from "./text-area";
 
-storiesOf("TextArea", module).add("Main", () => (
-	<div style={{ width: 300 }}>
-		<TextArea />
-        <DivPx size={8} />
-		<TextArea placeholder="Placeholder" />
-        <DivPx size={8} />
-		<TextArea readOnly value="Read-only" />
-        <DivPx size={8} />
-		<TextArea disabled />
-	</div>
-));
+export default {
+	title: "Components/TextArea",
+	component: TextArea,
```

Comment:
Should use Storybook's [controls](https://storybook.js.org/docs/react/essentials/controls) for simple props such as style, size, rows or disabled. Please see https://github.com/thien-do/moai/blob/master/core/src/components/date-input/date-input.stories.tsx#L12-L24 and https://github.com/thien-do/moai/blob/master/core/src/components/date-input/date-input.stories.tsx#L35-L50 for examplel